presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkBatchStreamReaders.java

```diff
@@ -232,6 +232,59 @@ private static String randomAsciiString(Random random)
         }
     }
 
+
+    static {
```

Comment:
Instead of using static code block, we should have different types in benchmark, and use JMH's warmup mode `BULK` or ` BULK_INDI` which is designed for this benchmark purpose. 

See the following benchmarks for examples:
- BenchmarkTransformKey
- BenchmarkTransformValue
- BenchmarkJsonToArrayCast
- BenchmarkJsonToMapCast

Basically the benchmark prepares for different types in different rounds. And in the benchmark wamrup, all of these types have been run to make sure JVM cannot optimize based on fixed type.

Reply:
@wenleix Thank you for the suggestion! I changed the code to
```
         Options options = new OptionsBuilder()
                 .verbosity(VerboseMode.NORMAL)
                 .include(".*" + BenchmarkBatchStreamReaders.class.getSimpleName() + ".*")
+                .warmupMode(WarmupMode.BULK)
                 .build();

```
Does it look correct?